data/blog/8-week-cadence.md

```diff
@@ -0,0 +1,58 @@
+---
+title: New Electron Release Cadence
+author: VerteDinde
+date: '2021-07-01'
+---
+
+Beginning in 2022, Electron will release a new major stable version every 8 weeks.
+
+---
+
+In 2019, Electron moved to a 12 week release cycle to match Chromium's 6 week release cycle. In the spring, Chromium announced that it would further speed up that cycle.
+
+Chrome now plans to release a new milestone every 4 weeks, starting with Chrome 94 in Q3 of 2021. In turn, the Electron team is updating our major release cadence to match. Beginning in 2022, Electron will release a new major stable version every 8 weeks.
```

Comment:
We should add this link
Chrome's release change announcement: https://blog.chromium.org/2021/03/speeding-up-release-cycle.html

Reply:
Would it make sense to say the first Electron version and date that will have an 8 week release cycle?
I think that's an important piece of information and it is at the end of the post.

Reply:
@sofianguy @molant Good points for both, will add the link and point out the first Electron version/date!